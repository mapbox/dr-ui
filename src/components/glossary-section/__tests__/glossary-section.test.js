import React from 'react';
import renderer from 'react-test-renderer';
import { testCases } from './glossary-section-test-cases';

describe('glossary-section', () => {
  describe(testCases.basic.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.basic;
      wrapper = renderer.create(
        React.createElement(testCase.component, testCase.props)
      );
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
