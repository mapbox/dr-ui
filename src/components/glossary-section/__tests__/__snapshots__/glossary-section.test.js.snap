// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`glossary-section Basic renders as expected 1`] = `
<div
  className="mb60"
>
  <h3
    className="ml18 mb6 txt-l color-gray"
  >
    C
  </h3>
  <div
    className="flex-parent flex-parent--wrap"
  >
    <div
      className="mb6 w240 flex-child"
    >
      <a
        className="color-gray-dark transition shadow-darken10-on-hover round clip inline-block w-full px12 py12 unprose"
        href="#chip"
        style={
          Object {
            "verticalAlign": "top",
          }
        }
      >
        <div
          className="px6 pb6"
        >
          <div
            className="mb6 txt-l"
          >
            Chip
          </div>
          <div
            className="txt-s color-gray"
          >
            A thin, crispy, and typically savory snack. It can be made of potatoes, corn, or a variety of other foods.
          </div>
        </div>
      </a>
    </div>
    <div
      className="mb6 w240 flex-child"
    >
      <a
        className="color-gray-dark transition shadow-darken10-on-hover round clip inline-block w-full px12 py12 unprose"
        href="#candy"
        style={
          Object {
            "verticalAlign": "top",
          }
        }
      >
        <div
          className="px6 pb6"
        >
          <div
            className="mb6 txt-l"
          >
            Candy
          </div>
          <div
            className="txt-s color-gray"
          >
            A sweet snack that is often fruity or chocolately or both!
          </div>
        </div>
      </a>
    </div>
  </div>
</div>
`;
